docs/fundamentals/code-analysis/quality-rules/il3001.md

```diff
@@ -20,7 +20,7 @@ f1_keywords:
 ## Cause
 
 When publishing as a single file (for example, by setting the PublishSingleFile property in a project to true), calling the `Assembly.GetFile(s)` methods for
-assemblies embedded inside the single-file bundle always throws an exception.
+assemblies embedded inside the single-file bundle always throws an exception, as these methods are not trim compatible.
```

Comment:
Sorry for the confusion, although Vitek mentioned that the wording safe/unsafe is something we want to avoid, the term trim compatible does not apply for single file, since trimming is an independent process that might or might not be executed for single-file apps. The term to document would be 'single-file compatible'